src/features/user/MFV2/ProfileLayout/ProfileLayout.module.scss

```diff
@@ -77,8 +77,11 @@
       border-radius: 24px;
     }
 
+    .mapContainer {
```

Comment:
Is there a reason why we've fixed the height for map container? Also, due to this the height of profile card will automatically be fixed to `450px`, would that not cause an issue for profile card with larger descriptions or have we set a character limit for the same?

Reply:
The height is fixed as the map will otherwise look squished if there is no description in the profile. The description itself is limited to 300 characters, so that is not creating an issue.